drivers/hud_apr/app/models/hud_apr/cell_details_concern.rb

```diff
@@ -0,0 +1,240 @@
+        'Question 27' => age_fields + household_fields + parenting_fields + gender_fields + health_fields + financial_fields + housing_fields + project_fields,
+      }
+      (common_fields + (extra_fields[question] || all_extra_fields)).uniq
+    end
+
+    def self.common_fields
+      [
+        :client_id,
+        :first_name,
+        :last_name,
+        :first_date_in_program,
+        :last_date_in_program,
+        :head_of_household,
+      ].freeze
+    end
+
+    def self.age_fields
+      [
+        :age,
+        :dob,
+      ].freeze
+    end
+
+    def self.parenting_fields
+      [
+        :parenting_youth,
+        :parenting_juvenile,
+      ].freeze
+    end
+
+    def self.veteran_fields
+      [
+        :veteran_status,
+      ].freeze
+    end
+
+    def self.homeless_fields
+      [
+        :chronically_homeless,
+        :date_homeless,
+        :times_homeless,
+        :months_homeless,
+      ].freeze
+    end
+
+    def self.pii_fields
+      [
+        :ssn,
+        :name_quality,
+        :dob_quality,
+        :ssn_quality,
+        :race,
+        :ethnicity,
+        :gender,
+        :gender_multi,
+      ].freeze
+    end
+
+    def self.universal_data_fields
+      [
+        :veteran_status,
+        :relationship_to_hoh,
+        # :client_location ?
+        :disabling_condition,
+      ].freeze
+    end
+
+    def self.financial_fields
+      [
+        :income_date_at_start,
+        :income_from_any_source_at_start,
+        :income_sources_at_start,
+        :income_date_at_annual_assessment,
+        :income_from_any_source_at_annual_assessment,
+        :income_sources_at_annual_assessment,
+        :income_date_at_exit,
+        :income_from_any_source_at_exit,
+        :income_sources_at_exit,
+        :income_total_at_start,
+        :income_total_at_annual_assessment,
+        :income_total_at_exit,
+        :non_cash_benefits_from_any_source_at_start,
+        :non_cash_benefits_from_any_source_at_annual_assessment,
+        :non_cash_benefits_from_any_source_at_exit,
+      ].freeze
+    end
+
+    def self.housing_fields
+      [
+        :destination,
+        :housing_assessment,
+        :prior_living_situation,
+      ].freeze
+    end
+
+    def self.project_fields
+      [
+        :project_type,
+        :project_tracking_method,
+      ].freeze
+    end
+
+    def self.timeliness_fields
+      [
+        # TODO: for Question 6, not sure what goes here
+      ].freeze
+    end
+
+    def self.inactive_records_fields
+      [
+        :date_of_last_bed_night,
+        :date_to_street,
+      ].freeze
+    end
+
+    def self.household_fields
+      [
+        :head_of_household,
+        :head_of_household_id,
+        :household_id,
+        :household_type,
+        :household_members,
+        :move_in_date,
+      ].freeze
+    end
+
+    def self.contacts_and_engagement_fields
+      [
+        # TODO: unclear to me what fields we need for question 9
```

Comment:
situations

Reply:
I don't see `situations` in the list of possible fields.

Reply:
Turns out that's because it's a table hanging off of APR client.  We don't currently expose that, so we'll just expose some basic household info.